packages/compass-aggregations/src/components/pipeline-builder-workspace/pipeline-as-text-workspace/index.tsx

```diff
@@ -16,19 +10,10 @@ import type { RootState } from '../../../modules';
 
 const containerStyles = css({
   display: 'flex',
-  marginLeft: spacing[3],
-  marginRight: spacing[3],
   height: '100%',
-
-  // align with stage editor design
-  border: `1px solid ${palette.gray.light2}`,
-  borderRadius: '4px',
-  boxShadow: `1px 1px 1px ${palette.gray.light2}`,
-});
-
-const containerDarkStyles = css({
-  borderColor: palette.gray.dark2,
-  boxShadow: `1px 1px 1px ${palette.gray.dark2}`,
+  borderBottomLeftRadius: 0,
+  borderBottomRightRadius: 0,
+  borderBottom: 'none',
```

Comment:
Hi Basit. This will remove the "box like" look from the text mode of pipeline builder. Just making sure that this is intentional?

Reply:
nice catch. totally missed it.